odoo/service/server.py

```diff
@@ -405,7 +405,8 @@ def cron_thread(self, number):
             time.sleep(SLEEP_INTERVAL + number)     # Steve Reich timing style
             registries = odoo.modules.registry.Registry.registries
             _logger.debug('cron%d polling for jobs', number)
-            for db_name, registry in registries.d.items():
+            # NB PA: List to avoid OrderedDict mutation during some tests. Unknown cause (as we don't mutate..)
```

Comment:
Might be worth adding this link to the comment, for context https://github.com/odoo/odoo/issues/79823